### PR TITLE
Change name to Percival

### LINKED ISSUE #24: Change name to Percival

### PR DESCRIPTION
**Pull request by abelsiqueira**
cf. @Egmara, @dpo

Also adds README (with a bunch of broken links) and LICENSE.

Closes #24

### CODE REVIEW

**Comment by abelsiqueira** [comment]
And after this, I think we can move it to JSO?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Percival.jl - An augmented Lagrangian solver
+
+[![Build Status](https://travis-ci.org/JuliaSmoothOptimizers/Percival.jl.svg?branch=master)](https://travis-ci.org/JuliaSmoothOptimizers/Percival.jl)
+[![Build status](https://ci.appveyor.com/api/projects/status/kp1o6ejuu6kgskvp/branch/master?svg=true)](https://ci.appveyor.com/project/dpo/percival-jl/branch/master)
+[![Build Status](https://api.cirrus-ci.com/github/JuliaSmoothOptimizers/Krylov.jl.svg)](https://cirrus-ci.com/github/JuliaSmoothOptimizers/Krylov.jl)
```
Comment (dpo) [inline README.md:5]: This says Krylov.jl.

Comment (abelsiqueira) [reply README.md]: Thanks. Helped find this: https://github.com/JuliaSmoothOptimizers/LinearOperators.jl/pull/150

**Comment by dpo** [comment]
Looks great. Could we see some preliminary results? Maybe some benchmarks against IPOPT to have an idea of where the code stands.

**Comment by abelsiqueira** [comment]
[results.pdf](https://github.com/CiDAMO/Percival.jl/files/4575056/results.pdf)
Constrained problems, up to 100 variables and constraints.

**Comment by abelsiqueira** [comment]
Looking into the smallest problem, `BURKEHAN`, looks like Percival is ill-equipped for infeasible problems.

**Comment by dpo** [comment]
Lots of humongous numbers in there. What's going on?

Also, there are evals of J?!

**Comment by abelsiqueira** [comment]
Thanks, Egmara.

Let's move the discussion to the slack channel.

**Pull request merged**